install_wfa.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Build WFA
+if [ ! -d wfa-source ]; then
+  git clone https://git.firesponse.es/external/vestec.git wfa-source
+  cd wfa-source
+  git checkout 8fce0a845836968e2658a8c64c58f39fba77c652
+  cd ..
+  mkdir wfa-build
+fi
+
+cd wfa-build
+cmake -DUSE_CATALYST=ON \
+      -DCMAKE_INSTALL_PREFIX=/home/d170/d170/shared/sw/fireapp/catalyst \
+      -DCMAKE_INSTALL_RPATH="\$ORIGIN/../lib:/home/d170/shared/sw/paraview/5.8.1-RC1/lib:/home/d170/shared/sw/gdal/3.1.0/lib" \
```

Comment:
`\$ORIGIN/../lib:/home/d170/shared/sw/paraview/5.8.1-RC1/lib` is not needed with the static adaptor lib.